LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 erh
+Copyright (c) 2023 Viam
```

Comment:
Anything we need to do for the license? I noticed other camera modules have Apache 2.0

Why MIT?

Reply:
He might have just added a random license, we should change this to Apache and change the date.

Reply:
OK will swap out for Apache

Reply:
Done